examples/js/loaders/GLTFLoader.js

```diff
@@ -3581,28 +3614,9 @@ THREE.GLTFLoader = ( function () {
 
 			if ( nodeDef.mesh !== undefined ) {
 
-				pending.push( parser.getDependency( 'mesh', nodeDef.mesh ).then( function ( mesh ) {
-
-					var node = parser._getNodeRef( parser.meshCache, nodeDef.mesh, mesh );
-
-					// if weights are provided on the node, override weights on the mesh.
-					if ( nodeDef.weights !== undefined ) {
-
-						node.traverse( function ( o ) {
+				pending.push( parser._invokeOne( function ( ext ) {
 
-							if ( ! o.isMesh ) return;
-
-							for ( var i = 0, il = nodeDef.weights.length; i < il; i ++ ) {
-
-								o.morphTargetInfluences[ i ] = nodeDef.weights[ i ];
-
-							}
-
-						} );
-
-					}
-
-					return node;
+					return ext.createNodeMesh && ext.createNodeMesh( nodeIndex );
 
```

Comment:
Sorry, I found a problem in this implementation after the review has been done. Plugins' createNodeMesh() should be called even if core `nodeDef.mesh` is undefined. e.g. GPU instancing extension without fallback in core. I would update soon.

Reply:
Updated